app/views/layouts/exercise_inputs/editors/_upload.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="form-group">
   <textarea id="solution_content" type="text" name="solution[content]" style="display: none"></textarea>
-  <input id="upload-input" type="file" class="upload submission-control" size="5120" accept=".<%= @exercise.language.extension %>" />
+  <input id="upload-input" type="file" class="upload submission-control" accept=".<%= @exercise.language.extension %>" />
```

Comment:
As the commit says, the `size` attribute does not have any effect on file-type inputs. See documentation at https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#htmlattrdefsize

Reply:
Nice catch. We should definitely enforce a DB limit, but we could also expose an attribute like `mu-upload-file-limit` here and use it in JS to reject an upload of a file bigger than that. What do u think?